Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/RecordNavPage.xaml.cs

```diff
@@ -40,197 +27,289 @@ namespace Ginger.BusinessFlowsLibNew.AddActionMenu
-
-        //            // AWTI.UIAElementInfo = AEI;
-        //            InitTree(AWTI);
-        //            break;
-        //        case AppWindow.eWindowType.AndroidDevice:
-        //            AndroidWindowTreeItem ADTI = new AndroidWindowTreeItem();
-
-        //            AndroidElementInfo AWI = new AndroidElementInfo();
-        //            AWI.WindowExplorer = mWindowExplorerDriver;
-        //            AWI.XPath = "/";
-        //            string pageSourceString2 = ((AndroidADBDriver)mWindowExplorerDriver).GetPageSource();
-        //            XmlDocument pageSourceXml2 = new XmlDocument();
-        //            pageSourceXml2.LoadXml(pageSourceString2);
-        //            AWI.XmlDoc = pageSourceXml2;
-        //            AWI.XmlNode = pageSourceXml2.SelectSingleNode("/hierarchy");
-
-        //            ADTI.AndroidElementInfo = AWI;
-        //            InitTree(ADTI);
-        //            break;
-        //        case AppWindow.eWindowType.Mainframe:
-        //            MainframeTreeItemBase MFTI = new MainframeTreeItemBase();
-        //            MFTI.Name = AW.Title;
-        //            MFTI.Path = AW.Path;
-        //            InitTree(MFTI);
-        //            break;
-        //        default:
-        //            Reporter.ToUser(eUserMsgKey.StaticErrorMessage, "Unknown Window type:" + AW.WindowType);
-        //            break;
-        //    }
-
-        //    //if (WindowControlsGridView.Visibility == System.Windows.Visibility.Visible)
-        //    //{
-        //    //    RefreshControlsGrid();
-        //    //}
-        //}
-        //void InitTree(ITreeViewItem RootItem)
-        //{
-        //    //WindowControlsTreeView.Tree.ClearTreeItems();
-        //    //mRootItem = RootItem;
-
-        //    //mTreeRootItem = WindowControlsTreeView.Tree.AddItem(RootItem);
-        //    //mTreeRootItem.IsExpanded = false;
-        //}
-
-        //private void RefreshWindowsButton_Click(object sender, RoutedEventArgs e)
-        //{
-
-        //}
-
-        //private void AddSwitchWindowActionButton_Click(object sender, RoutedEventArgs e)
-        //{
-
-        //}
-    }
+                xRecordingButton.ButtonText = "Start Recording";
+                xRecordingButton.ToolTip = "Start Recording";
+                xRecordingButton.ButtonImageType = eImageType.Play;
+            }
+            xRecordingButton.ButtonStyle = (Style)FindResource("$RoundTextAndImageButtonStyle_Execution");            
+        }
+
+        SingleItemTreeViewSelectionPage mApplicationPOMSelectionPage = null;
+        ObservableList<POMBindingObjectHelper> PomModels = new ObservableList<POMBindingObjectHelper>();
+        
+        private void MAppModelSelectionPage_SelectionDone(object sender, SelectionTreeEventArgs e)
+        {
+            AddSelectedBuinessFlows(e.SelectedItems);
+        }
+
+        private void AddSelectedBuinessFlows(List<object> selectedPOMs)
```

Comment:
AddSelectedBuinessFlows - method name is not correct

Reply:
Changed name